.github/workflows/build-ios.yml

```
@@ -0,0 +1,73 @@
+name: Build iOS App
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'mobile/**'
+      - 'shared/**'
+  push_tags:
```

actionlint (1.7.12)

[error] unknown Webhook event "push_tags". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)